File: aat-lib/src/main/java/ch/bailu/aat_lib/preferences/presets/SolidTrackerAutopause.java
package ch.bailu.aat_lib.preferences.presets;

import ch.bailu.aat_lib.preferences.SolidAutopause;
import ch.bailu.aat_lib.preferences.StorageInterface;
import ch.bailu.aat_lib.resources.Res;

public class SolidTrackerAutopause extends SolidAutopause {
    private final static String KEY="autopause";

    public SolidTrackerAutopause(StorageInterface s, int i) {
        super(s, KEY, i);
    }

    @Override
    public String getLabel() {
        return Res.str().p_tracker_autopause();
    }


}

File: aat-lib/src/main/java/ch/bailu/aat_lib/preferences/presets/SolidTrackerAutopause.kt
package ch.bailu.aat_lib.preferences.presets

import ch.bailu.aat_lib.preferences.SolidAutopause
import ch.bailu.aat_lib.preferences.StorageInterface
import ch.bailu.aat_lib.resources.Res

class SolidTrackerAutopause(storage: StorageInterface, index: Int) : SolidAutopause(
    storage, KEY, index) {
    override fun getLabel(): String {
        return Res.str().p_tracker_autopause()
    }

    companion object {
        private const val KEY = "autopause"
    }
}
